ccm-connector/.gitignore

```diff
@@ -0,0 +1,19 @@
+/bin
+.settings
+.project
+.classpath
+.gradle
+.idea
+*.iml
+.apib
+.DS_Store
+*.jks
+atlassian*
+userdatas/
+out/
+curl_config_admin_user
+*.suml
+startit.sh
+build/
+test-output/
+**/generated/
```

Comment:
if we exclude generated, it will cause idea issues, won't it?

Reply:
Good catch! I did this before somebody pointed the same thing out on a different review.